boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dts

```diff
@@ -0,0 +1,223 @@
+	status = "okay";
+	pinctrl-0 = <&i2s0_default>;
+	pinctrl-1 = <&i2s0_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+&uart0 {
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart0_default>;
+	pinctrl-1 = <&uart0_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+&i2c1 {
+	compatible = "nordic,nrf-twim";
+	status = "okay";
+	pinctrl-0 = <&i2c1_default>;
+	pinctrl-1 = <&i2c1_sleep>;
+	pinctrl-names = "default", "sleep";
+
+	vbat_sensor: ina231@44 {
+		compatible = "ti,ina230";
+		reg = <0x44>;
+		config = <INA230_CONFIG(INA230_OPER_MODE_SHUNT_BUS_VOLTAGE_CONT,
+					INA230_CONV_TIME_4156,
+					INA230_CONV_TIME_4156,
+					INA230_AVG_MODE_1024)>;
+		current-lsb-microamps = <1>;
+		rshunt-milliohms = <510>;
+	};
+
+	vdd1_codec_sensor: ina231@45 {
+		compatible = "ti,ina230";
+		reg = <0x45>;
+		config = <INA230_CONFIG(INA230_OPER_MODE_SHUNT_BUS_VOLTAGE_CONT,
+					INA230_CONV_TIME_4156,
+					INA230_CONV_TIME_4156,
+					INA230_AVG_MODE_1024)>;
+		current-lsb-microamps = <1>;
+		rshunt-milliohms = <2200>;
+	};
+
+	vdd2_codec_sensor: ina231@41 {
+		compatible = "ti,ina230";
+		reg = <0x41>;
+		config = <INA230_CONFIG(INA230_OPER_MODE_SHUNT_BUS_VOLTAGE_CONT,
+					INA230_CONV_TIME_4156,
+					INA230_CONV_TIME_4156,
+					INA230_AVG_MODE_1024)>;
+		current-lsb-microamps = <1>;
+		rshunt-milliohms = <2200>;
+	};
+
+	vdd2_nrf_sensor: ina231@40 {
+		compatible = "ti,ina230";
+		reg = <0x40>;
+		config = <INA230_CONFIG(INA230_OPER_MODE_SHUNT_BUS_VOLTAGE_CONT,
+					INA230_CONV_TIME_4156,
+					INA230_CONV_TIME_4156,
+					INA230_AVG_MODE_1024)>;
+		current-lsb-microamps = <1>;
+		rshunt-milliohms = <1000>;
+	};
+};
+
+&spi4 {
+	compatible = "nordic,nrf-spim";
+	status = "okay";
+	cs-gpios = <&gpio0 11 GPIO_ACTIVE_LOW>, <&gpio0 17 GPIO_ACTIVE_LOW>;
+	pinctrl-0 = <&spi4_default>;
+	pinctrl-1 = <&spi4_sleep>;
+	pinctrl-names = "default", "sleep";
+	sdhc0: sdhc@0 {
+		compatible = "zephyr,sdhc-spi-slot";
+		reg = <0>;
+		status = "okay";
+		sdmmc {
+			compatible = "zephyr,sdmmc-disk";
+			status = "okay";
+		};
+
+		spi-max-frequency = <8000000>;
+	};
+	cs47l63: cs47l63@1 {
+		reg = <1>;
+	};
+};
+
+&pwm0 {
+	status = "okay";
+};
+
+&timer0 {
+	status = "okay";
+};
+
+&timer1 {
+	status = "okay";
+};
+
+&timer2 {
+	status = "okay";
+};
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 0x00010000>;
+		};
+		slot0_partition: partition@10000 {
+			label = "image-0";
+		};
+		slot0_ns_partition: partition@50000 {
+			label = "image-0-nonsecure";
+		};
+		slot1_partition: partition@80000 {
+			label = "image-1";
+		};
+		slot1_ns_partition: partition@c0000 {
+			label = "image-1-nonsecure";
+		};
+		storage_partition: partition@fa000 {
+			label = "storage";
+			reg = <0x000fa000 0x00006000>;
+		};
+	};
+};
+
+zephyr_udc0: &usbd {
+	compatible = "nordic,nrf-usbd";
+	status = "okay";
+};
```

Comment:
This is a needless copy of: https://github.com/zephyrproject-rtos/zephyr/blob/69fb9991ca48e2fd3b74d3f30bc227e575797920/boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dts#L205-L207.